image/templates/helm/shared/templates/_metadata.tpl

```diff
@@ -63,7 +91,7 @@
 {{- end -}}
 
 {{/*
-  srox._annotations $annotations $ $objType $objName $forPod
+  srox._annotations $ $resultingAnnotations $annotations $objType $objName $forPod
 
   Writes all applicable [pod] annotations (including default annotations) for
   $objType/$objName into $annotations. Pod labels are written iff $forPod is true.
```

Comment:
Is this really still true? Hasn't the role of `$annotations` now changed to providing defaults, and the output parameter is now `$resultingAnnotations`?

Perhaps it would make the diff smaller if you kept the old name, and named the new one `$extraAnnotations` or similar?

I may be just babbling jibberish, but in case my suspicion is correct, please also carefully adjust descriptions of other functions you changed.

Reply:
I have renamed the variables to keep the PR shorter and I have updated the template comments.